### ISSUE TITLE
DeepLabCut ,  No Images Created when extracting frames

### ISSUE THREAD

**Issue by ReetKaur15** [DeepLabCut/DeepLabCut#2301]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

No Images created, when trying to extract frames. When I press the Extract Frames button, popup arise with message that frames are extracted of video of interest. But no frames created. 

I have also tried to Run DLC as an admin. Still, same issue arise. 

Please Give your suggestions to solve this issue. 

### Operating System

operating system = Windows 11

### DeepLabCut version

dlc version = 2.3.5

### DeepLabCut mode

single animal

### Device type

CPU

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by n-poulsen** [comment]
Hi @ReetKaur15, are any logs/errors printed to the prompt you're using?

**Comment by ReetKaur15** [comment]
Hi @n-poulsen , there are no errors printed, I get following logs even after extracting frames. Also, get pop up on GUI that frames are extracted of interested video. But, No images generated. 


```
(deeplabcut) PS C:\ProgramData> python -m deeplabcut
C:\ProgramData\anaconda3\envs\deeplabcut\lib\site-packages\numpy\_distributor_init.py:30: UserWarning: loaded more than 1 DLL from .libs:
C:\ProgramData\anaconda3\envs\deeplabcut\lib\site-packages\numpy\.libs\libopenblas.FB5AE2TYXYH2IJRDKGDGQ3XBKLKTF43H.gfortran-win_amd64.dll
C:\ProgramData\anaconda3\envs\deeplabcut\lib\site-packages\numpy\.libs\libopenblas64__v0.3.21-gcc_10_3_0.dll
  warnings.warn("loaded more than 1 DLL from .libs:"
Loading DLC 2.3.5...
Starting GUI...
```

Also, When I tried this using jupyter notebook, then it works fine. But with GUI, It is not extracting frames. 
Could you please find out why this is happening?

Many Thanks in advance :)

**Comment by n-poulsen** [comment]
@ReetKaur15 Could you provide us with the exact parameters you're using to extract frames in the GUI vs the ones you use in the jupyter notebook?

**Comment by ReetKaur15** [comment]
@n-poulsen, Yes please find below the parameters I have used with both GUI and jupyter notebook settings. 

**With GUI:**

Extraction Method= Automatic
GUI slider Width= 25
Extraction Algorithm = Kmeans or Uniform 
Frame Cropping= disabled
Cluster step=1

**With Jupyter Notebook** 

```python
deeplabcut.extract_frames(config_path, mode='automatic', algo='kmeans', userfeedback=False, crop=False)
```

**Comment by TEKAP0** [comment]
I met the same problem, and i'd like to know the latest solution. THX！

**Comment by ReetKaur15** [comment]
Hi @TEKAP0 , have you tried using jupyter notebook?

**Comment by KonradDanielewski** [comment]
Are you using the `Select videos` button to choose specific videos? There is an issue with it on Windows

**Comment by ReetKaur15** [comment]
@KonradDanielewski , yes I am using the _**select videos**_ button to choose specific video using windows. 
Is there any solution to this ? Please let me know if you have any update regarding this. 

Many Thanks and regards,

**Comment by KonradDanielewski** [comment]
> @KonradDanielewski , yes I am using the _**select videos**_ button to choose specific video using windows. Is there any solution to this ? Please let me know if you have any update regarding this.
> 
> Many Thanks and regards,

This will fix the behavior on Windows https://github.com/DeepLabCut/DeepLabCut/pull/2286. 

If you're extracting from all videos in the project you don't need to select them, just click on the extract button - it will run in the same way as the line you've run in jupyter

**Comment by ReetKaur15** [comment]
> This will fix the behavior on Windows #2286.
> 
> If you're extracting from all videos in the project you don't need to select them, just click on the extract button - it will run in the same way as the line you've run in jupyter

@KonradDanielewski , Thanks a lot, now it is extracting the frames. 
But, when I try to label the frames using GUI, no images displayed in labeled data, folder is empty. When I check the folder manually then , frames are present in the folder. How to solve this ?

**Comment by KonradDanielewski** [comment]
There is some strange behavior happening on Windows when using napari with admin privileges (drag&drop doesn't work, folders don't open automatically etc.). You can open the folder with images from the drop down menu after you open the napari labelling GUI or just run the terminal without admin privileges - napari should work as expected then. 

I'm still not sure what leads to this issue on Windows. If anything, I'd expect it to be the other way where it would happen if you're not an admin.

**Comment by n-poulsen** [comment]
Fixed with #2286
